src/describe.cpp

```diff
@@ -9,7 +30,7 @@ int main(int argc, char** argv)
     Omega_h::Mesh mesh = Omega_h::read_mesh_file(argv[1], lib.world());
 
     auto verbose = false;
-    if (argc == 3) verbose = (std::string(argv[2]) == "on");
+    if (argc >= 3) verbose = (std::string(argv[2]) == "on");
```

Comment:
Let's try to use `Omega_h_cmdline.hpp` to define and parse user inputs.  See `exo2osh.cpp` for one example.  Using these APIs should give us a help (`-h`) option.